Tutorial Series/Introduction to Financial Python/Tutorial12 Modern Portfolio Theory.html

```diff
@@ -96,3 +98,9 @@
 What if I extend the the market capital line to achieve a higher return at a higher risk? There is another import assumption that there is no transaction cost, which means I can borrow money as much as I wish. In order to achieve a higher return, we need to borrow risk-free asset, or short risk-free, and invest the proceeds together with our own money to the market portfolio. Under such circumstance, our weight on risk-free is negative, and the weight on the market portfolio is larger than 1.
 <h2>Summary</h2>
 In this chapter we learned modern portfolio theory. It's important to understand here is that by holding difference stocks we can achieve a portfolio with higher sharpe ratio, and all the investors holds the same amount of risky asset, which is the market portfolio. In the next chapter we will introduce asset pricing with CAPM, which is the most classical asset pricing model.
+<h2>Algorithm:</h2>
+Mean-variance analysis is usually used to optimize portfolios with several strategies. Here we treat Dow 30 stocks as strategy and designed an algorithm to test mean-variance analysis:
+
+&lt;script src='https://www.quantconnect.com/terminal/backtest.js?sid=71761bc8ce01e7c81b5ea91f54a82640'&gt;&lt;/script&gt;
```

Comment:
This is not correct, @ShallweXiaowei  you are pasting from the wordpress escaped html this line should be:
`<script src='https://www.quantconnect.com/terminal/backtest.js?sid=71761bc8ce01e7c81b5ea91f54a82640' ></script>`